doc/api/child_process.md

```diff
@@ -849,6 +849,10 @@ added: v0.5.9
 The `'message'` event is triggered when a child process uses [`process.send()`][]
 to send messages.
 
+*Note*: since the message goes through JSON serialization and parsing, it might
+not be the same as what is originally sent, see notes of
+[the `JSON.stringify()` specification][`JSON.stringify` spec].
```

Comment:
Nit: The comma after `sent` should be a period. `See notes at...` can be its own sentence.